source/_docs/cookies.md

```diff
@@ -6,7 +6,7 @@ categories: []
 ---
 
 ## Cookie Naming Requirements
-For cookies to function on cached pages they must match the pattern: `STYXKEY[a-zA-Z0-9_-]`.
+For cookies to function on cached pages they must match the pattern: `STYXKEY[a-zA-Z0-9_-]`. Note that on WordPress sites, if you want session cookies to be excluded from caching, they must be named with `wp-` as their prefix.
```

Comment:
Is this a recent infrastructure change? I know that WordPress core cookies have a `wp-` prefix but I thought Varnish whitelisted them explicitly.

Reply:
For context, we had a Give customer report a problem with sessions on Pantheon. I confirmed the problem in a new Pantheon environment. We had a Pantheon support technician make the suggestion that Give using `wp_` as the prefix was the problem, I tested that hypothesis by changing our prefix and it resolved the problem both on my new environment and for the customer. So, in my experience between all of those points of contact it's definitely the case. But I can't speak to internal documented changes that Pantheon might have. Thanks for reviewing!

Reply:
Here's our Give issues in this regard:
* https://github.com/impress-org/give/issues/3819
* https://github.com/impress-org/give/issues/3705#issuecomment-423485726

Reply:
I think two types of cookies are being conflated here:

- STYXKEY cookies are [cache-varying](https://pantheon.io/docs/cookies/#cache-varying-cookies).
- Cookies with the `wp-[A-Za-z0-9_]+` prefix are [cache-busting](https://pantheon.io/docs/caching-advanced-topics/#cookie-handling). There are several other cookie prefixes that bust cache.

There are Varnish rules for both types of these cookies, but they accomplish different things.

I'd suggest:
- Remove or clarify this "Cookie Naming Requirements" section since it only refers to STYXKEY-style cookies.
- Link or copy the cache-busting cookie naming patterns from the Advanced Caching doc ([this](https://pantheon.io/docs/caching-advanced-topics/#cookie-handling)) over to the Cookies doc.

Reply:
Yes, 👍 on @sarahg's suggestion; the cache-busting cookies should not be described in the same section as the cache-varying cookies. (At the moment only STYXKEY is cache-varying, but there may be more of these in the future). Deleting the section called "Cookie Naming Requirements" makes sense, as this information is duplicated further down on the page. Referencing the Advanced Caching page as suggested is also a good idea.